Tests/DecodableTests.swift

```diff
@@ -43,7 +43,7 @@ class DecodableTestCase: BaseTestCase {
             XCTAssertEqual(testObject.subInt, -1, "test object subInt does not match expected value")
             XCTAssertEqual(testObject.subString, "sub test string", "test object subString does not match expected value")
         } catch {
-            XCTFail("Parser unexpectedly failed by throwing error: \(error)")
```

Comment:
I certainly agree. The reason we generally use `Test encountered unexpected error: \(error)` is that it is generic enough that it works in all cases. It will always be 100% accurate no matter how many times we copy-paste it.

I went ahead and made that change b/c there's no parser call in this test and it felt a bit misleading. Generally, if this test would ever fail, I would quickly look for the `try` and pair that up with the error message. IMO the fact that `Parser` is listed doesn't really help me diagnose anything about the failure itself. Hence the change.

I can certainly modify the message if you'd like. I'd recommend leaving it as is though given the number of places I've found outdated comments and documentation.